test/es-module/test-esm-loader-chaining.mjs

```diff
@@ -101,6 +101,28 @@ describe('ESM: loader chaining', { concurrency: true }, () => {
     assert.strictEqual(code, 0);
   });
 
+  it('should accept only the correct arguments', async () => {
+    const { stdout } = await spawnPromisified(
+      execPath,
+      [
+        '--loader',
+        fixtures.fileURL('es-module-loaders', 'loader-log-args.mjs'),
+        '--loader',
+        fixtures.fileURL('es-module-loaders', 'loader-with-too-many-args.mjs'),
+        ...commonArgs,
+      ],
+      { encoding: 'utf8' },
+    );
+
+    assert.match(stdout, /resolve arg count: 3/);
+    assert.match(stdout, /specifier: 'node:fs'/);
```

Comment:
```suggestion
    assert.match(stdout, /^resolve arg count: 3$/m);
    assert.match(stdout, /^specifier: 'node:fs'$/m);
```
So it doesn't match in case arg count is `30` for some reason.